examples/grid.cpp

```diff
@@ -31,8 +31,6 @@
 #include <Yami.h>
```

Comment:
Maybe you can separate this change from the patch set and get it merged individually.